zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/socket/Message.java

```diff
@@ -191,7 +191,13 @@
     INTERPRETER_INSTALL_RESULT,   // [s-c] Status of an interpreter installation
     COLLABORATIVE_MODE_STATUS,    // [s-c] collaborative mode status
     PATCH_PARAGRAPH,              // [c-s][s-c] patch editor text
-    NOTICE                        // [s-c] Notice
+    NOTICE,                       // [s-c] Notice
+    REMOVE_SELECTED_PARAGRAPHS,   // [c-s] remove selected paragraphs
```

Comment:
Can we merge these with the single paragraph operation to reduce code duplication ? Otherwise if we introduce new operation in future, we have to introduce 2 message, one for single paragraph, another for multiple paragraph, which is inefficient.

Reply:
Thanks. I'll think about how to improve this.